src/components/app/Root.js

```diff
@@ -28,21 +32,28 @@ export class Root extends PureComponent<RootProps> {
   render() {
     const { store } = this.props;
     return (
-      <ErrorBoundary message="Uh oh, some error happened in profiler.firefox.com.">
+      <NonLocalizedErrorBoundary message="Uh oh, some error happened in profiler.firefox.com">
         <Provider store={store}>
           <AppLocalizationProvider>
-            <DragAndDrop>
-              <UrlManager>
-                <ServiceWorkerManager />
-                <ProfileLoader />
-                <AppViewRouter />
-                <FooterLinks />
-                <WindowTitle />
-              </UrlManager>
-            </DragAndDrop>
+            <Localized
+              id="Root--error-boundary-message"
+              attrs={{ message: true }}
+            >
+              <LocalizedErrorBoundary message="Uh oh, some error happened in profiler.firefox.com">
```

Comment:
Here I added another one so that the non localized version would be only for errors in AppLocalizationProvider.